go/vt/vttablet/tabletserver/tx_engine.go

```diff
@@ -349,10 +349,10 @@ func (te *TxEngine) Init() error {
 // all previously prepared transactions from the redo log.
 // this should only be called when the state is already locked
 func (te *TxEngine) open() {
-	te.txPool.Open(te.env.DBConfigs().AppWithDB(), te.env.DBConfigs().DbaWithDB(), te.env.DBConfigs().AppDebugWithDB())
+	te.txPool.Open(te.env.Config().DB.AppWithDB(), te.env.Config().DB.DbaWithDB(), te.env.Config().DB.AppDebugWithDB())
 
 	if te.twopcEnabled && te.state == AcceptingReadAndWrite {
-		te.twoPC.Open(te.env.DBConfigs())
+		te.twoPC.Open(te.env.Config().DB)
```

Comment:
I am not liking these kind of calls Config().DB.AppWithDB. I like it before with DBConfig().func

Reply:
I can add a convenience function that shortcuts `Config().DB` to `DBConfigs()` keeping it backward compatible. Will that work?

Reply:
That will work